src/routes/stats/components/StatsLeaderboardCell.vue

```diff
@@ -80,6 +81,10 @@ export default {
       default: '',
     },
   },
+  setup() {
+    const { t } = useI18n();
+    return { t };
+  },
```

Comment:
I think I found a setting to avoid needing to do this. We may be able to use https://vue-i18n.intlify.dev/api/general.html#globalinjection @itsalaidbacklife